resources/docker/Dockerfile

```diff
@@ -54,6 +54,8 @@ ENV SERVER_URL ''
 
 ENV ISSUES ''
 
+ENV SECURE ''
```

Comment:
Please provide a default value for this environment variable. Thanks!

Reply:
The default value is specified in the docker compose yaml

Reply:
Should be specified here also.

Reply:
Wouldn't that make it no longer an environmental value, but just a hard-coded value that can't be changed?

Reply:
No. That's not how ENV in docker works. It just specifies default values within the Docker image so if an image is spun up without passing any Environment Variable it will use those values in ENV.

Putting it in docker-compose is just like passing in an environment variable. Docker image should be self contained and should be able to spin up a docker container without any values getting passed in, in the ideal world.

Reply:
Here's a resource on it: https://docs.docker.com/engine/reference/builder/#env

Reply:
OK that makes sense.